interp/builtin.go

```diff
@@ -19,6 +19,12 @@ import (
 	"mvdan.cc/sh/v3/syntax"
 )
 
+// optStatuses are the option statuses' text display
+var optStatuses = map[bool]string{
```

Comment:
nit: make this a function with an if statement, there isn't really a need for a map for just two entries.